extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/querykit/groupby/GroupByPostShuffleFrameProcessor.java

```diff
@@ -306,17 +321,23 @@ private static HavingSpec cloneHavingSpec(final GroupByQuery query)
    * this processor. Kept in sync with the signature generated by {@link GroupByQueryKit}.
    */
   private static VirtualColumns makeVirtualColumnsForFrameWriter(
+      @Nullable final VirtualColumn partitionBoostVirtualColumn,
       final ObjectMapper jsonMapper,
       final GroupByQuery query
   )
   {
+    List<VirtualColumn> virtualColumns = new ArrayList<>();
```

Comment:
We need a UT for this.

Reply:
I removed the conditionals all together, therefore the code is now straightforward.